packages/esm-patient-chart-app/src/banner/patient-banner.scss

```diff
@@ -0,0 +1,80 @@
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.container {
+  border-top: 0.063rem solid $ui-03;
+  border-bottom: 0.063rem solid $ui-03;
+  background-color: $ui-01;
+}
+
+.patientBanner {
+  display: flex;
+}
+
+.patientName {
+  @include type.type-style("heading-03");
+  margin-right: 0.25rem;
+}
+
+.patientAvatar {
+  width: 5rem;
+  height: 5rem;
+  margin: 1rem;
+  border-radius: 1px;
+}
+
+.patientAvatarButton {
+  cursor: pointer;
+  border: none;
+  padding: 0px;
+  background: none;
+}
+
+.patientInfo {
+  width: 100%;
+}
+
+.demographics {
+  @include type.type-style("body-compact-02");
+  color: $text-02;
+  margin-top: 0.375rem;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.patientNameRow {
+  margin-top: 0.875rem;
+}
+
+.flexRow {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+}
+
+.identifiers {
+  @include type.type-style("body-compact-02");
+  color: $ui-04;
+}
+
+.tooltipPadding {
+  padding: 0.25rem;
+}
+
+.tooltipSmallText {
+  font-size: 80%;
+}
+
+.actionsButtonText {
+  @include type.type-style("body-compact-01");
+  color: $interactive-01;
+}
+
+:global(.cds--tooltip--definition .cds--tooltip__trigger){
```

Comment:
This is dangerous. We need to clean all of these up
New Ticket -> https://issues.openmrs.org/browse/O3-1492

Reply:
okay I will check it out too.